CHANGELOG.md

```diff
@@ -1,5 +1,72 @@
 # Liana daemon and GUI release notes
 
+## 5.0
+
+This release introduces support for the Coldcard signing device, experimental Taproot support and a
+number of user experience improvements.
+
+### Features
+
+#### Liana daemon / library
+
+- Experimental support for Taproot was added.
+- It's now possible to configure the connection to `bitcoind` using a user and a password in place
+  of the cookie file. See the discussion at https://github.com/wizardsardine/liana/issues/356.
+- The `getinfo` result now contains the "descriptor's timestamp": that is the oldest date at which
+  we scanned the blockchain for coins.
+- The `createspend` command now doesn't error anymore on insufficient funds, it instead returns the
+  missing amount in its result.
+- The `listspendtxs` command now accepts a new optional parameter to filter the result by txids.
+
+#### Liana GUI
+
+- Support for Coldcard was added. At the time of writing Miniscript support on the Coldcard is only
+  available on the Edge firmware.
+- You can now choose to create a Taproot descriptor in the installer. See
+  [doc/signing_devices.md](doc/signing_devices.md) for details about the compatibility with hardware
+  wallets.
+- When creating a Spend transaction, one can now set the amount for any recipient to "Max" to sweep
+  all the funds from the wallet to this recipient, minus the amount set for any other recipient.
+- The automated coin selection when creating a spend transaction now also considers unconfirmed
+  coins.
+- When creating a Spend transaction, you can now change screen and come back to your draft. The
+  state does not get cleared as soon as you leave the "Spend" menu anymore.
+- We now display warnings to the user when creating a Spend transaction. For instance when the
+  change output's value was too small that we added it to fees.
+- RBF transaction now get automatically labeled.
+- Signing a transaction on a hardware device does not hide the transaction details anymore.
+- When broadcasting a transaction, the payments are now immediately available on the home page.
+- The QR code for deposit address is now shown in a pop-up, which makes it available for all new
+  addresses a user may generate on the Deposit screen.
+- The address QR codes now also contain the derivation index in the URI.
+- We now display a warning if a user tries to RBF a transaction whose change output is being spent
```

Comment:
There's also a warning when broadcasting a spend in case any of the inputs are currently being spent by another transaction.